src/test/java/io/stargate/sgv2/jsonapi/api/v1/NamespaceResourceIntegrationTest.java

```diff
@@ -91,10 +91,9 @@ public void invalidNamespaceName() {
           .post(NamespaceResource.BASE_PATH, "7_no_leading_number")
           .then()
           .statusCode(200)
-          .body(
-              "errors[0].message",
-              startsWith("Request invalid, the field postCommand.namespace not valid"))
-          .body("errors[0].exceptionClass", is("ConstraintViolationException"));
+          .body("errors[0].exceptionClass", is("ConstraintViolationException"))
+          .body("errors[0].message", startsWith("Request invalid: field 'namespace' value"))
+          .body("errors[0].message", containsString("value \"7_no_leading_number\" not valid"));
```

Comment:
This is the major fix right?
nitpick, reason of not combine these two messages into one?

Reply:
Right, that is the fix.

And yes, I think I should just combine the parts together since it is covering the whole String anyway.